media/css/cms/rich-text.scss

```
@@ -3,50 +3,10 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 @use '~@mozilla-protocol/core/protocol/css/includes/lib' as *;
+@import '~@mozilla-protocol/core/protocol/css/base/utilities/rich-text';
```

```suggestion
@use '~@mozilla-protocol/core/protocol/css/base/utilities/rich-text';
```

trying to remove `@import` if possible (only if it's an easy swap)